core/src/main/java/org/elasticsearch/index/seqno/LocalCheckpointService.java

```diff
@@ -32,33 +33,41 @@
  */
 public class LocalCheckpointService extends AbstractIndexShardComponent {
 
+    public static String MAX_SEQ_NO = "max_seq_no";
```

Comment:
same comment about location of these. Also, they miss a final on them. (same goes for `GLOBAL_CHECKPOINT_KEY`)